src/os_io_task.c

```diff
@@ -33,18 +33,26 @@ void io_start(void) {
 void bolos_init_ios(void) {
```

Comment:
I' didn't know this exists... this is somehow a duplicate of what's done in lib_standard_app/main.c: common_app_init(). Maybe we can merge?